.tekton/pull-request.yaml

```diff
@@ -74,6 +74,25 @@ spec:
                 python -m pip install gitlint
                 git fetch --unshallow
                 gitlint --commits "origin/$(params.target-branch)..HEAD"
+      - name: yaml-lint
+        workspaces:
+          - name: workspace
+            workspace: workspace
+        runAfter:
+          - clone-repository
+        taskSpec:
+          workspaces:
+            - name: workspace
+          steps:
+            - name: yaml-lint
+              image: registry.access.redhat.com/ubi9/python-39:latest
+              workingDir: $(workspaces.workspace.path)
+              script: |
+                #!/bin/bash -ex
+
+                python -m pip install yamllint
+                cd $(workspaces.workspace.path)
+                yamllint .
```

Comment:
Will it check every yaml on every sub directory?

Reply:
yes

Reply:
ok. The guidelines will be something to be added to the contributing guide.

Also, when it comes to Prometheus unit tests, I didn't find a way yet to break long lines, and they tend to be very long. We need to either find a way to do it, or find a way to exempt them specifically from line length tests.

Reply:
yeah. I will also add the rule files for yamllint

Reply:
cool

Reply:
Removed the long-line errors